src/armonik_cli/commands/common.py

```diff
@@ -0,0 +1,123 @@
+import re
+
+import rich_click as click
+
+from datetime import timedelta
+from typing import cast, Tuple, Union
+
+
+endpoint_option = click.option(
+    "-e",
+    "--endpoint",
+    type=str,
+    required=True,
+    help="Endpoint of the cluster to connect to.",
+    metavar="ENDPOINT",
+)
+output_option = click.option(
+    "-o",
+    "--output",
+    type=click.Choice(["yaml", "json", "table"], case_sensitive=False),
+    default="json",
+    show_default=True,
+    help="Commands output format.",
+    metavar="FORMAT",
+)
+debug_option = click.option(
+    "--debug", is_flag=True, default=False, help="Print debug logs and internal errors."
+)
+
+
+class KeyValuePairParam(click.ParamType):
+    """
+    A custom Click parameter type that parses a key-value pair in the format "key=value".
+
+    Attributes:
+        name (str): The name of the parameter type, used by Click.
```

Comment:
soit tu utilises le typing soit tu documentes dans la docstring mais pas un mélange des deux. (et cela est un commentaire général)

Reply:
I agree, I'll make these changes over the next few pr.